pom.xml

```diff
@@ -64,5 +88,52 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-cps</artifactId>
+            <version>${workflow-cps.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>2.11.1</version>
```

Comment:
pipeline deps should be `optional`... whilst I have no reason not to install pipeline, just because I use junit I should not require it.

Reply:
I thought about that, but decided we're far enough along in the Pipeline era now that it's eminently reasonable to assume that Pipeline be installed everywhere. =)

Reply:
I agree with @jtnord , you should not mark as not optional dependency only because you assume pipeline is installed everywhere because with this trend jenkins became a big monolite where installing one plugin downloads all.

Reply:
@abayer NO NO.  ping me offline if you want to be educated why pipeline will never be installed on ALL jenkins instances, and why doing this in junit is pure evil.